README.md

```diff
@@ -0,0 +1,45 @@
+# scylla-cdc-rust
+Scylla-cdc-rust is a library that enables consuming the 
+[Scylla Change Data Capture Log](https://docs.scylladb.com/using-scylla/cdc/)
+in Rust applications. 
+The library automatically and transparently handles errors and topology changes 
+of the underlying Scylla cluster. 
+Thanks to that, the API allows the user to read the CDC log without having to deeply understand 
+the internal structure of CDC.
+
+It is recommended to get familiar with the documentation of CDC first, 
+in order to understand the concept: 
+<https://docs.scylladb.com/using-scylla/cdc/>
+
+The library was written in pure Rust,
+using [Scylla Rust Driver](https://crates.io/crates/scylla)
+and [Tokio](https://crates.io/crates/tokio).
+
+## Getting started
+The best place to get started with the library is the [tutorial](tutorial.md). 
+You can also check the [Scylla University lessons](https://university.scylladb.com/courses/scylla-operations/lessons/change-data-capture-cdc/)
+to learn more about the CDC.
+
+## Examples
+The repository also contains two exemplary applications:
+
+- [Printer](scylla-cdc-printer): prints all changes from CDC log for a selected table.
+- [Replicator](scylla-cdc-replicator): replicates a table from one Scylla cluster to another by reading the CDC log.
+
+## Contact
+TODO
+
```

Comment:
What should we put here? Scylla-cdc-java directs to the `#cdc` channel on the slack.

Reply:
Yes, it is a good idea to direct to the same slack channel here.